buffer.go

```diff
@@ -91,6 +92,11 @@ func (r *recordBuffer) Add(m *router.Message) error {
 		return err
 	}
 
+	// We default to a uuid if the template didn't match.
+	if pKey == "" {
+		pKey = uuid.New()
```

Comment:
I think I will log everything by default and introduce a `KINESIS_SILENCE` variable to shut it up (which we will set to true in our cluster)